update docs after legacy configs have been removed (#17634)
apache/druid: docs/development/extensions-core/approximate-histograms.md

### Null handling

> Null values will be tracked in the `missingValueCount` field of the histogram.

Reviewer: ```suggestion
Druid tracks null values in the `missingValueCount` field of the histogram.
```
Nit: avoid future tense
